tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -277,11 +296,12 @@ def check_data(self, peercluster, imagespec):
         peercluster.wait_for_status(imagespec=imagespec, state_pattern="up+replaying")
         peercluster.wait_for_replay_complete(imagespec)
         export_path = "/home/cephuser/image.export"
-
+        time.sleep(300)
```

Comment:
Can we avoid blind sleeps? It adds time to execution... if there is no mechanism to determine whether to move forward or not, then adding a comment on why we are sleeping would be helpful. This reduces the maintenance effort...